packages/client/src/aiClient.ts

```diff
@@ -77,9 +82,9 @@ export default class AIClient {
     });
   }
 
-  panic(error: Error): never {
+  panic(error: Error): void {
     this.disconnect();
-    throw error;
+    this.callbacks.onError?.(error);
```

Comment:
@dustinbyrne throwing this error was causing the RPC server to exit, so I changed it to call the `onError` callback. Is that OK?